CONTRIBUTING.md

```diff
@@ -85,5 +85,18 @@ Use the [`BREAKING CHANGE:` notation](https://www.conventionalcommits.org/en/v1.
 
 ## Code Style
 
+### Styling
+
 Please always run `black` before submitting a pull request. You can install
-`black` with `pip install black`.
+`black` with `pip install black`. The GitHub's Actions which serve as CI/CD will, alongside running pytest, also check this before enabling a Pull request to be merged.
+
+### Documentation
+
+Do not forget to document your code - functions and classes, using docstrings, since our tool will be used as a library in other projects, and such documentation is very useful to programmers. For functions always be sure to state parameters, return value and short description of the funcionality itself (even though that should be already clear from the name itself, see below). For classes, please include descriptions of attributes, methods, and the class itself.
```

Comment:
The construct "code - functions and classes" is not entirely clear. Do you mean "code, functions, and classes" or "code (its functions and classes)"

Reply:
Maybe: comment your code and document its functions and classes :)